src/System.Globalization/tests/CompareInfo/CompareInfoCompare.cs

```diff
@@ -561,6 +513,18 @@ public void Test148()
         [Fact]
         public void Test156() { Test(CultureInfo.InvariantCulture, new string('a', 5555), new string('a', 5555), 0, CompareOptions.None); }
 
+        [Fact]
```

Comment:
LGTM.

Note that I have a low pri issue to change some tests to more descriptive test names instead of things like PostTest2 or Test156 etc. You may want to consider that going forward.